### PR TITLE
Fix failing blacken-docs pre-commit hook

### PR DESCRIPTION
**Pull request by hukkin**

### CODE REVIEW

**Comment by abravalheri** [comment]
Thank you very much @hukkin.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -39,10 +39,10 @@ repos:
     language_version: python3
 
 - repo: https://github.com/asottile/blacken-docs
-  rev: v1.11.0
+  rev: v1.12.1
   hooks:
   - id: blacken-docs
-    additional_dependencies: [black]
+    additional_dependencies: [black==22.1.0]
```
Comment (abravalheri) [inline .pre-commit-config.yaml:45]: Is there any incompatibility with some versions, or you are just following the recommendations from the [docs](https://github.com/asottile/blacken-docs#usage-with-pre-commit)?

Now that `black` is becoming stable, with a clear update/deprecation policy, could we rely on the latest version?

Comment (hukkin) [reply .pre-commit-config.yaml:45]: I'm just following the recommendation from the docs. Note that if we don't pin black version, then we should probably just delete this `additional_dependencies` line. `black` is required by `blacken-docs` anyways, just not version pinned.

Black's policy still allows formatting changes at the start of a new year, so one sensible alternative is pinning `black==22.*`.

Comment (abravalheri) [reply .pre-commit-config.yaml:45]: Thank you very much for the clarification!

**Comment by abravalheri** [comment]
Thanks @hukkin, I merged your PR into master and I will adopt your suggestion about `black==22.*` in a separated commit.

**Pull request merged**